user-guide/Troubleshooting/Procedures/Investigating_Web_Issues.md

```diff
@@ -107,6 +108,21 @@ GQI recording is a debugging feature that allows you to save GQI communication a
 > [!NOTE]
 > Recording might impact performance as data is written to the disk. Once the recording has been made, stop the recording by deleting the folder.
 
+#### GQI DxM repair
+
+If you encounter issues with GQI DxM, for example, if you have [enabled the GQI DxM](xref:GQI_DxM#enabling-the-use-of-the-gqi-dxm) but are unable to see GQI DxM version on the *About* page, try repairing the service by running the GQI DxM installer:
+
+1. Using Task Manager, stop the DataMiner GQI service.
+
+   > [!NOTE]
+   > Make sure the associated DataMiner GQI process is stopped as well.
+
+1. Navigate to `C:\Skyline DataMiner\Tools\ModuleInstallers\Web`.
+
+1. Run the *DataMiner GQI* DxM installer in Repair mode.
+
+1. Restart the web server using IIS Manager to reconnect the web apps with the GQI DxM.
```

Comment:
Could you include more information on how to do this? Some users may not be familiar with IIS Manager and will need clear step-by-step instructions for this.